html-aam/accname-computation-by-element/tr-td-th.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Accname computation tests for &lt;tr&gt;</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <script src="/html-aam/scripts/html-aam-accname-utils.js"></script>
+</head>
+
+<body>
+  <script>
+    const testConfig = {
+      urlSpec: 'https://www.w3.org/TR/html-aam-1.0/#tr-td-th-elements-accessible-name-computation',
+      elements: {
+        'tr': { parents: ['table', 'tbody'] },
+        'th': { parents: ['table', 'thead', 'tr'] },
+        'td': { parents: ['table', 'tbody', 'tr'] },
```

Comment:
Seems like the `parents` parameter should be `ancestors`? Is there a way to add any attributes to the ancestors?

Reply:
> Seems like the parents parameter should be ancestors?

Sigh, yes, that would have been much more accurate. 🤦🏻‍♂️  I’ve [renamed](https://github.com/web-platform-tests/wpt/pull/44965/commits/05325043d4cce5731fb2792136c95df0449e484c) `parents` to `ancestors` and `children` to `descendants`.

> Is there a way to add any attributes to the ancestors?

Not currently. I did consider it along the way, possibly using [selector parsing](https://github.com/web-platform-tests/wpt/pull/44965/files#diff-de7b88e6abae2f964cd620997ad7a41d3027fc9e65745964bd2da4b9f3a940b9R287-R291) like I’m already doing with the dictionary key for the test element itself. 🤔